lib/constants.js

```diff
@@ -42,6 +42,8 @@ if (process.env.TRITONTEST_CLI_CONFIG_DIR) {
      * machine-specific data dir.
      */
     CLI_CONFIG_DIR = mod_path.resolve(process.env.APPDATA, 'Joyent', 'Triton');
+} else if (process.env.XDG_CONFIG_HOME) {
+    CLI_CONFIG_DIR = mod_path.resolve(process.env.XDG_CONFIG_HOME, '.triton');
```

Comment:
Just noticed this while testing with my XDG setup, this should be
```
CLI_CONFIG_DIR = mod_path.resolve(process.env.XDG_CONFIG_HOME, 'triton');
```
without the `.`, because `.config` is the hidden `XDG_CONFIG_HOME` directory usually, then all sub-directories get non-hidden names.

Reply:
Can you file an issue for this?

Reply:
Like here: https://github.com/joyent/node-triton/pull/291/files#diff-602f56bb692d9bf3bdc5fef3d75d5fa33fe841c3d21ca46294bb6fdaf0785d0cR54

Reply:
Sure will do

Reply:
Thank you for the quick response